src/gatling/resources/conf/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+  <root level="${logLevel}">
+    <appender-ref ref="CONSOLE"/>
+  </root>
+
+  <logger name="io.netty" level="INFO"/>
+  <logger name="io.gatling" level="INFO"/>
+  <logger name="io.gatling.http.fetch.ResourceFetcherActor" level="WARN"/>
```

Comment:
Would it make sense to log any other gatling files on a lower level like debug?

Reply:
I think it is okay currently, easy to change as needed. That one is on WARN because it was annoying

Reply:
👍